database/migrations/2024_08_29_124400_create_media_temporary_uploads_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('media_temporary_uploads', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('created_by_user_id')->index()->constrained(app(config('auth.providers.users.model'))->tableName())->cascadeOnDelete();
```

Comment:
`->tableName()` is not available on the User model by default. So you need to use the method available by laravel, which is `->getTable()`